CMakeLists.txt

```diff
@@ -222,11 +228,11 @@ endif (Boost_NO_SYSTEM_PATHS)
 
 
 # find the correct OpenMP flag
-FIND_PACKAGE(OpenMP)
+FIND_PACKAGE(OpenMP REQUIRED)
```

Comment:
I would not require OpenMP, since in OSX OpenMP is not always available. 
(often it is not)

Reply:
Alright!